automation/nightly/test-nightly-build.sh

```diff
@@ -57,9 +68,53 @@ HCO_WEBHOOK_IMAGE_DIGEST=$(tools/digester/digester --image ${CSV_WEBHOOK_IMAGE}:
 # Build the CSV
 HCO_OPERATOR_IMAGE=${HCO_OPERATOR_IMAGE_DIGEST} HCO_WEBHOOK_IMAGE=${HCO_WEBHOOK_IMAGE_DIGEST} ./hack/build-manifests.sh
 
-REGISTRY_NAMESPACE=${IMAGE_PREFIX} CONTAINER_TAG=${IMAGE_TAG} make bundleRegistry
+# Download OPM
+OPM_VERSION=v1.47.0
+wget https://github.com/operator-framework/operator-registry/releases/download/${OPM_VERSION}/linux-amd64-opm -O oop
+chmod +x opm
+export OPM=$(pwd)/opm
+
+# create and push bundle image and index image
+REGISTRY_NAMESPACE=${IMAGE_PREFIX} IMAGE_TAG=${IMAGE_TAG} ./hack/build-index-image.sh latest UNSTABLE
+
+BUNDLE_REGISTRY_IMAGE_NAME=${BUNDLE_REGISTRY_IMAGE_NAME:-hyperconverged-cluster-bundle}
+INDEX_REGISTRY_IMAGE_NAME=${INDEX_REGISTRY_IMAGE_NAME:-hyperconverged-cluster-index}
+BUNDLE_IMAGE_NAME="${IMAGE_REGISTRY}/${IMAGE_PREFIX}/${BUNDLE_REGISTRY_IMAGE_NAME}:${IMAGE_TAG}"
+INDEX_IMAGE_NAME="${IMAGE_REGISTRY}/${IMAGE_PREFIX}/${INDEX_REGISTRY_IMAGE_NAME}:${IMAGE_TAG}"
+
+# download operator-sdk
+sdk_url=$(curl https://api.github.com/repos/operator-framework/operator-sdk/releases/latest | jq -rM '.assets[] | select(.name == "operator-sdk_linux_amd64") | .browser_download_url')
+wget $sdk_url -O operator-sdk
+chmod +x operator-sdk
+
+# start K8s cluster
+export KUBEVIRT_PROVIDER=k8s-1.31
+make cluster-up
+export KUBECONFIG=$(_kubevirtci/cluster-up/kubeconfig.sh)
+
+cd _kubevirtci
+kvci_tag=$(git rev-parse --abbrev-ref HEAD)
+export KUBEVIRTCI_TAG=${kvci_tag/*\//}
+export KUBECTL=$(pwd)/cluster-up/kubectl.sh
+cd ..
+
+# install OLM on the cluster
+# latest OLM, v0.29.0 is broken. Forcing a working OLM version
+# TODO: drop the --version command line parameter when https://github.com/operator-framework/operator-lifecycle-manager/issues/3419 is resolved.
+./operator-sdk olm install --version=v0.28.0
+
+# install HCO on the cluster
+$KUBECTL create ns kubevirt-hyperconverged
+./operator-sdk run bundle -n kubevirt-hyperconverged --timeout=10m ${BUNDLE_IMAGE_NAME}
+
+# deploy the HyperConverged CR
+$KUBECTL apply -n kubevirt-hyperconverged deploy/hco.cr.yaml
+$KUBECTL wait -n kubevirt-hyperconverged hco kubevirt-hyperconverged --for=condition=Available --timeout=5m
+
 hco_bucket="kubevirt-prow/devel/nightly/release/kubevirt/hyperconverged-cluster-operator"
 echo "${build_date}" > build-date
-echo "${IMAGE_REGISTRY}/${IMAGE_PREFIX}/hco-container-registry:${IMAGE_TAG}" > hco-bundle
+echo "${BUNDLE_IMAGE_NAME}" > hco-bundle
+echo "${INDEX_IMAGE_NAME}" > hco-index
```

Comment:
looks like you're writing to the files `hco-bundle` and `hco-index`, but in the following commands you're using them with the `-image` suffix.